airbyte-integrations/bases/source-acceptance-test/unit_tests/test_test_full_refresh.py

```diff
@@ -108,17 +126,109 @@ def record_message_from_record(record_: Dict) -> List[AirbyteMessage]:
             )
 
 
-def get_default_catalog(schema):
-    return ConfiguredAirbyteCatalog(
-        streams=[
-            ConfiguredAirbyteStream(
-                stream=AirbyteStream(
-                    name="test_stream",
-                    json_schema=schema,
-                    supported_sync_modes=["full_refresh"],
-                ),
-                sync_mode="full_refresh",
-                destination_sync_mode="overwrite",
+recordset_comparison_test_cases = [
+    (
+        [["id"]],
+        [{"id": 1, "first_name": "Thomas", "last_name": "Edison"}, {"id": 2, "first_name": "Nicola", "last_name": "Tesla"}],
+        [{"id": 1, "first_name": "Albert", "last_name": "Einstein"}, {"id": 2, "first_name": "Joseph", "last_name": "Lagrange"}],
+        False,
+        "pk_sets_equal_success",
+    ),
```

Comment:
`test_case_name` is only used for readability isn't it? I suggest leverage pytest params id:
```suggestion
    pytest.params(
        [["id"]],
        [{"id": 1, "first_name": "Thomas", "last_name": "Edison"}, {"id": 2, "first_name": "Nicola", "last_name": "Tesla"}],
        [{"id": 1, "first_name": "Albert", "last_name": "Einstein"}, {"id": 2, "first_name": "Joseph", "last_name": "Lagrange"}],
        False,
        id="pk_sets_equal_success",
    ),
```

and not build the id afterward in the list comprehension.